packages/themes/styles/scss/lib/bootstrap/_bootstrap.scss

```diff
@@ -13,7 +13,8 @@
 //@import "~bootstrap-sass/assets/stylesheets/bootstrap/glyphicons";
 
 // Core CSS
-@import "~bootstrap-sass/assets/stylesheets/bootstrap/scaffolding";
+//@import "~bootstrap-sass/assets/stylesheets/bootstrap/scaffolding";
```

Comment:
remove commented out line? or is this useful to keep as a reference?

Reply:
This is following the "standard" practice in this file. Allows us to easily see when we overrode an underlying file.